frontend/views/components/modal/ModalBaseTemplate.vue

```diff
@@ -52,11 +52,11 @@ export default ({
     justify-content: flex-start;
     align-items: center;
     background: $general_2;
-    padding: 0 1rem;
+    // padding: 0 1rem;
 
-    @include tablet {
-      padding: 0 1.5rem;
-    }
+    // @include tablet {
+    //   padding: 0 1.5rem;
+    // }
```

Comment:
@pieer this needs your review

Reply:
If there is no need for padding, then this can be removed.